serving/samples/autoscale-go/README.md

```diff
@@ -46,57 +45,158 @@ A demonstration of the autoscaling capabilities of a Knative Serving Revision.
-   Total: 1706     Inflight: 300   Done: 555       Success Rate: 100.00%   Avg Latency: 0.4794 sec
-   Total: 2334     Inflight: 264   Done: 628       Success Rate: 100.00%   Avg Latency: 0.5207 sec
-   Total: 2911     Inflight: 300   Done: 577       Success Rate: 100.00%   Avg Latency: 0.4401 sec
-   ...
+   Summary:
+     Total:        30.3379 secs
+     Slowest:      0.7433 secs
+     Fastest:      0.1672 secs
+     Average:      0.2778 secs
+     Requests/sec: 178.7861
+
+     Total data:   542038 bytes
+     Size/request: 99 bytes
+
+   Response time histogram:
+     0.167 [1]     |
+     0.225 [1462]  |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
+     0.282 [1303]  |■■■■■■■■■■■■■■■■■■■■■■■■■■■■
+     0.340 [1894]  |■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
+     0.398 [471]   |■■■■■■■■■■
+     0.455 [159]   |■■■
+     0.513 [68]    |■
+     0.570 [18]    |
+     0.628 [14]    |
+     0.686 [21]    |
+     0.743 [13]    |
+
+
+   Latency distribution:
+     10% in 0.1805 secs
+     25% in 0.2197 secs
+     50% in 0.2801 secs
+     75% in 0.3129 secs
+     90% in 0.3596 secs
+     95% in 0.4020 secs
+     99% in 0.5457 secs
+
+   Details (average, fastest, slowest):
+     DNS+dialup:   0.0007 secs, 0.1672 secs, 0.7433 secs
+     DNS-lookup:   0.0000 secs, 0.0000 secs, 0.0000 secs
+     req write:    0.0001 secs, 0.0000 secs, 0.0045 secs
+     resp wait:    0.2766 secs, 0.1669 secs, 0.6633 secs
+     resp read:    0.0002 secs, 0.0000 secs, 0.0065 secs
+
+   Status code distribution:
+     [200] 5424 responses
+
+
+
+   NAME                                             READY   STATUS    RESTARTS   AGE
+   autoscale-go-00001-deployment-78cdc67bf4-2w4sk   3/3     Running   0          26s
+   autoscale-go-00001-deployment-78cdc67bf4-dd2zb   3/3     Running   0          24s
+   autoscale-go-00001-deployment-78cdc67bf4-pg55p   3/3     Running   0          18s
+   autoscale-go-00001-deployment-78cdc67bf4-q8bf9   3/3     Running   0          1m
+   autoscale-go-00001-deployment-78cdc67bf4-thjbq   3/3     Running   0          26s
    ```
 
-   > Note: Use CTRL+C to exit the load test.
-
-1. Watch the Knative Serving deployment pod count increase.
-   ```
-   kubectl get deploy --watch
-   ```
-   > Note: Use CTRL+C to exit watch mode.
-
 ## Analysis
 
 ### Algorithm
 
 Knative Serving autoscaling is based on the average number of in-flight requests
 per pod (concurrency). The system has a default
-[target concurrency of 100.0](https://github.com/knative/serving/blob/3f00c39e289ed4bfb84019131651c2e4ea660ab5/config/config-autoscaler.yaml#L35).
+[target concurrency of 100.0](https://github.com/knative/serving/blob/3f00c39e289ed4bfb84019131651c2e4ea660ab5/config/config-autoscaler.yaml#L35) but [we used 10](service.yaml#L26) for our service.  We loaded the service with 50 concurrent requests so the autoscaler created 5 pods (`50 concurrent requests / target of 10 = 5 pods`)
```

Comment:
nit: might be worth updating the first link to a relative link for consistency.

Reply:
Is that the right thing to do between repos?  A relative link would be good within the same repo because it would stay within the same branch etc...

Reply:
Ah you are right. Thanks.